Don't install the i18n folder; it's pointless to do so since the compiled
translations are in the locale folder.

--- a/setup.py
+++ b/setup.py
@@ -256,7 +256,7 @@
         packages.extend(['hgext.inotify', 'hgext.inotify.linux'])
 
 datafiles = []
-for root in ('templates', 'i18n', 'help'):
+for root in ('templates', 'help'):
     for dir, dirs, files in os.walk(root):
         dirs[:] = [x for x in dirs if not x.startswith('.')]
         files = [x for x in files if not x.startswith('.')]
